src/databricks/labs/ucx/source_code/graph.py

```diff
@@ -303,28 +306,45 @@ class MaybeDependency:
     problems: list[DependencyProblem]
 
 
+# the DependencyResolver is responsible for locating "stuff", mimicking the Databricks runtime behavior
```

Comment:
Please move this to the docstring of the calss

Reply:
done